web/templates/popovers/left_sidebar/left_sidebar_stream_actions_popover.hbs

```diff
@@ -9,40 +9,40 @@
             <span class="popover-stream-name">{{stream.name}}</span>
         </li>
         <li role="separator" class="popover-menu-separator"></li>
-        <li role="none" class="link-item popover-menu-inner-list-item hidden-for-spectators">
```

Comment:
@opmkumar the renaming should be in it's own separate commit.

Reply:
Done. Thanks!